src/reduxStore.jsx

```diff
@@ -1,6 +1,8 @@
 import { createStore, applyMiddleware, combineReducers } from 'redux';
 import thunk from 'redux-thunk';
-import reducer from './reducer';
+import Transactions from './Stats/reducer';
 
-const mainReducer = combineReducers({ reducer });
+const mainReducer = combineReducers({
+  Transactions,
```

Comment:
- Reducers are methods, we should use camelCase.
- Previously I meant we should rename the stats folder to transactions

Reply:
fixed